quickwit-cli/tests/cli.rs

```diff
@@ -50,6 +53,21 @@ fn create_logs_index(test_env: &TestEnv) {
     .success();
 }
 
+fn ingest_docs_with_options(input_path: &Path, test_env: &TestEnv, options: &str) {
```

Comment:
can you let this function be called from `ingest_docs` and add the assertions from there

Reply:
you mean to call `ingest_docs_with_options` from `ingest_docs`? but `ingest_docs` doesn't take an extra `options` parameter to pass in the `--clean-cache`, unless you mean something else.

Reply:
I meant to unduplicate the code from `ingest_docs(input_path: &Path, test_env: &TestEnv` by calling `ingest_docs_with_options`